### ISSUE TITLE
Search returns empty, even the searched contents is present

### ISSUE THREAD

**Issue by spirillen** [jellyfin/jellyfin-web#6316]
**Describe The Bug**
Read title and Please see screenshots, it says it all

**Steps To Reproduce**

1. Search
2. In example we search for `Trigger Warning`
3. The results returns nothing

**Expected Behavior**
The film would have been shown

**Logs**
Not showing anything on issue

**Screenshots**
Search result

![Image](https://github.com/user-attachments/assets/78aa5130-c29f-490b-928d-087c8884f5b0)

The Film is present

![Image](https://github.com/user-attachments/assets/aa39394b-4ab6-4db2-a9e3-0c533830e7af)

**System (please complete the following information):**
 - Platform: Linux (Debian Bookworm (12))
 - Browser: Firefox-esr
 - Jellyfin Version: 10.10.1

**Additional Context**

**Comment by viown** [comment]
Any errors in Dashboard -> Logs after search?

**Comment by spirillen** [comment]
> Any errors in Dashboard -> Logs after search?

Nope, that is what is weird, it feels like a timeout, but I can't find anything in the log(s)

And here is even the search without any search...

![Image](https://github.com/user-attachments/assets/dc4043ad-d7d8-4a14-a818-f1fa6b4f169a)

This is the log for when I made this reply and retested

```log
[2024-11-12 11:53:43.062 +01:00] [ERR] Error processing request: "Could not find file '/etc/jellyfin/users/Tablet/profile.png'". URL "GET" "/Users/45e6ce0efb884db696b9a9332d3e6b6e/Images/Primary".
[2024-11-12 11:54:20.951 +01:00] [INF] Authentication request for "USER" has succeeded.
[2024-11-12 11:54:20.952 +01:00] [INF] Current/Max sessions for user "USER": 1/0
[2024-11-12 11:54:20.955 +01:00] [INF] Logging out access token "ID"
[2024-11-12 11:54:20.987 +01:00] [INF] Creating new access token for user
```

**Comment by viown** [comment]
That is odd. Do you still have issues with duplicate media? Do you see duplicate logs like in https://github.com/jellyfin/jellyfin-web/issues/6302? These could be related issues, and if we can figure out what's causing it, we can open an issue in https://github.com/jellyfin/jellyfin/issues

**Comment by DunklerPhoenix** [comment]
I often have the same problem with my large library and it seems that a new search is triggered with any new entered letter in the search field. If you type to fast or delete a letter the cpu spikes up and the search doen't show any results. I ignored this because I thought sqlite could be overwhelmed with the amount of data and I wait for the support of other db types, but it would be great if we can get an optional  option to switch from instant search to a "trigger on search icon" as server-wide option :)

**Comment by spirillen** [comment]
I doubt this is to be put on the SQLite alone, but sure, I would vote for an upgrade to PGSql or MariaDB.

But this is happening on both HP ML 350G6 8 core/20GB & HP DL380G8 32 core/100GB hardware with SSD drives in hardware raid

**Comment by viown** [comment]
> I often have the same problem with my large library and it seems that a new search is triggered with any new entered letter in the search field. If you type too fast or delete a letter the server cpu spikes up and the search doesn't show any results. I ignored this because I thought sqlite could be overwhelmed with the amount of data and I wait for the support of other db types, but it would be great if we can get an optional option to switch from instant search to a "trigger on search icon" search as server-wide option :)

Searches should be [debounced](https://github.com/jellyfin/jellyfin-web/blob/2ecab9052a98b86b35135f96dc7d968f9d9fd739/src/apps/stable/routes/search.tsx#L22), but you seem to be correct, I'm seeing a lot of search requests per keystroke which I don't remember was there before 🤔

**Comment by viown** [comment]
https://github.com/jellyfin/jellyfin-web/pull/6593 should help significantly improve the performance of searches as it reduces the number of queries as well as runs them synchronously rather than asynchronously.

**Comment by spirillen** [comment]
Thanks for the update @viown
